lms/static/sass/elements/_controls.scss

```diff
@@ -416,3 +416,83 @@
       color: $link-color;
     }
   }
+
+  %btn-secondary-white {
+  @extend %ui-btn-secondary !optional;
```

Comment:
@marcotuts made `!optional` otherwise need to copy dependancies from studio, also `btn-secondary-white` is not being used in dialog.